enterprise_access/apps/core/signals.py

```diff
@@ -0,0 +1,39 @@
+"""
+Signal handlers for core app.
+"""
+import logging
+
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from enterprise_access.apps.content_assignments.models import LearnerContentAssignment
```

Comment:
What would you think of moving this file into the `content_assignments` app instead?  That dependency direction feels a little more obvious to me.